src/app/tasks-arr/tasks-arr.component.html

```diff
@@ -0,0 +1,27 @@
+<div [ngSwitch]="role">
```

Comment:
try to have meaningful name of your component and try to avoid using "arr" as we can simply rename it as task.component or tasks.component